playbooks/compute/nto/roles/configurecluster/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Ensure python packages are installed
```

Comment:
Doesn't this part of the collection requirements?

Reply:
yes - But this is for enabling this role to be run separately